static/main.js

```diff
@@ -0,0 +1,91 @@
+$(document).ready(function () {
+  $('.add_entry').on('submit', function(event){
+      event.preventDefault();
+      add_post();
+    });
+});
+
+
+function add_post() {
+    var title = $('#title').val();
+    var text = $('#text').val();
+    $.ajax({
+      url: '/new',
+      type: 'POST',
+      dataType: 'json',
+      data: {'title': title, 'text': text},
+      success: success
+    });
+}
+
+function open_edit() {
+    var id = $('.entry').attr('id').split("entry")[1];
```

Comment:
This is a brittle way to acquire the ID of the entry to be edited.  Can you think of a better way to handle it?